src/init.cpp

```diff
@@ -871,8 +871,6 @@ bool AppInitParameterInteraction(const ArgsManager& args)
     nUserMaxConnections = args.GetArg("-maxconnections", DEFAULT_MAX_PEER_CONNECTIONS);
     nMaxConnections = std::max(nUserMaxConnections, 0);
 
-    // Trim requested connection counts, to fit into system limitations
-    // <int> in std::min<int>(...) to work around FreeBSD compilation issue described in #2695
```

Comment:
> Both are outdated since #14336.

Why? The code the comment referred to previously was just moved a few lines down (unchanged) in #14336. This PR should have either moved the comment down, or had an actual explanation of why the FreeBSD compilation issue is no longer relevant.

Reply:
:man_facepalming: 

ouch, you are right...